### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by SteveMieskoski**
### Feature

- [x] Updated CHANGELOG.md
- [x] Add PR label

### CODE REVIEW

**Comment by kvhnuke** [comment]
@SteveMieskoski there are so many commented lines and the checklist is no complete

File: src/builds/configs/base-routes.js [hunk 1]
```diff
@@ -146,7 +164,17 @@ const router = [
         path: 'verify-message',
         name: 'Verify Message',
         component: VerifyMessageContainer
+      },
+      {
+        path: 'nft-manager',
+        name: 'NFTManager',
+        component: NFTManagerContainer
       }
+      // {
```
Comment (gamalielhere) [inline src/builds/configs/base-routes.js]: comments

File: src/layouts/InterfaceLayout/components/InterfaceAds/InterfaceAds.scss [hunk 2]
```diff
@@ -0,0 +1,129 @@
+@import '~@/scss/GlobalVariables';
+
+//.bottom-image-container {
```
Comment (gamalielhere) [inline src/layouts/InterfaceLayout/components/InterfaceAds/InterfaceAds.scss]: comments

File: src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue [hunk 3]
```diff
@@ -0,0 +1,549 @@
...
+      tokenHelper: {},
+      imagesRetrieved: false,
+      showDetails: false,
+      selectedContract: '0x06012c8cf97bead5deae237070f9587f8e7a266d',
+      detailsFor: {},
+      nftTokens: {},
+      nftData: {},
+      ownedTokens: [],
+      tokenContractAddress: '0xeA3352C1a3480Ac5a32Fcd1F2854529BA7193F14',
+      useDevAddress: true,
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue]: remove comments and I believe this is `false`

File: src/layouts/InterfaceLayout/components/InterfaceAds/InterfaceAds.scss [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      color: #0b2840;
+      .inactive {
+        opacity: 0.2;
+      }
+      .active {
+        opacity: 1;
+      }
+    }
+  }
+  //img {
```
Comment (gamalielhere) [inline src/layouts/InterfaceLayout/components/InterfaceAds/InterfaceAds.scss]: moar comments

File: src/layouts/InterfaceLayout/components/InterfaceAds/adComponents/index.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import buyEthAd from './buyEthAd';
+import staticAd from './staticAd';
+
+//
```
Comment (gamalielhere) [inline src/layouts/InterfaceLayout/components/InterfaceAds/adComponents/index.js]: comments

File: src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue (hunk 3 above)
Comment (kvhnuke) [reply src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue]: also remove references to `useDevAddress`

File: src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue [hunk 6]
```diff
@@ -0,0 +1,549 @@
...
+        `${URL_BASE}?contract=${contract}&token=${token}`,
+        {
+          mode: 'cors',
+          cache: 'no-cache',
+          method: 'GET'
+        }
+      );
+      const hex = await image.text();
+      return hex;
+      // return 'data:' + response.headers['content-type'] + ';base64,'
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue]: remove comment

File: src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue [hunk 7]
```diff
@@ -0,0 +1,549 @@
...
+
+        const result = await this.getOwnedTokens(
+          supportedNftTokens,
+          address,
+          nftData
+        );
+        this.ready = true;
+        this.processing = false;
+        this.imagesRetrieved = true;
+        this.$forceUpdate();
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue]: I dont think you need this!

File: src/layouts/InterfaceLayout/containers/NFTManagerContainer/containers/NftDetails/NftDetails.vue [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+  mounted() {
+    this.ERC721tokenContract = new this.web3.eth.Contract(ERC721);
+  },
+  methods: {
+    prepareTransfer(toAddress) {
+      this.toAddress = toAddress;
+      this.ERC721tokenContract.options.address = this.nft.contract;
+    },
+    buildData() {
+      if (
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/NFTManagerContainer/containers/NftDetails/NftDetails.vue]: cant we just use `transferFrom` which is common in all 721 including crypto kitties

File: src/layouts/InterfaceLayout/containers/NFTManagerContainer/containers/NftDetails/NftDetails.vue [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+        this.nft.contract.toLowerCase() ===
+        '0x06012c8cf97bead5deae237070f9587f8e7a266d'.toLowerCase()
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/NFTManagerContainer/containers/NftDetails/NftDetails.vue]: dont hardcode addresses

File: src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue [hunk 10]
```diff
@@ -0,0 +1,493 @@
...
+      tokenHelper: {},
+      imagesRetrieved: true,
+      showDetails: false,
+      selectedContract: '0x06012c8cf97bead5deae237070f9587f8e7a266d',
+      detailsFor: {},
+      nftTokens: {},
+      nftData: {},
+      ownedTokens: [],
+      tokenContractAddress: '0xeA3352C1a3480Ac5a32Fcd1F2854529BA7193F14',
+      useDevAddress: false, // todo remove dev item
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue]: please remove all references to `useDevAddress`, you should simply have a global accountAddress object which you can switch between `devAddress` and localAccount

File: src/layouts/InterfaceLayout/containers/NFTManagerContainer/containers/NftDetails/NftDetails.vue [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+    };
+  },
+
+  computed: {
+    ...mapState([
+      'account',
+      'gasPrice',
+      'web3',
+      'network',
+      'linkQuery',
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/NFTManagerContainer/containers/NftDetails/NftDetails.vue]: `linkQuery` not used

File: src/layouts/InterfaceLayout/containers/DashboardContainer/DashboardContainer.vue [hunk 12]
```diff
@@ -0,0 +1,304 @@
...
+    };
+  },
+
+  computed: {
+    ...mapState([
+      'account',
+      'gasPrice',
+      'web3',
+      'network',
+      'linkQuery',
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/DashboardContainer/DashboardContainer.vue]: not used, please remove all unused mapStates

**Pull request merged**